demo/js/components/ComponentExample/component-example.scss

```diff
@@ -33,7 +33,7 @@
     color: $color__navy-gray-6;
     font-size: 0.75rem;
     padding: 0.5rem 1rem;
-    z-index: 9999;
+    z-index: 8000;
```

Comment:
Should we use `z()` with `modal`? https://github.com/IBM/carbon-components/blob/master/src/globals/scss/_layout.scss#L82

Reply:
this is for the component example on our demo site, so instead of `modal` I will use `overlay`

Reply:
Oh duh, this is what I get for only look at diff view. Totally thought it was for modal 😂 My bad!